vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -68,7 +68,7 @@
       </Annotation>
 
 
-      <Term Name="ApplicationTimeSupport" Type="Temporal.ApplicationTimeSupportType" Nullable="false" AppliesTo="Collection">
+      <Term Name="ApplicationTimeSupport" Type="Temporal.ApplicationTimeSupportType" Nullable="false" AppliesTo="Collection EntityType">
```

Comment:
Why should we allow `EntityType` here?

Reply:
At the moment, annotation if entity types is _implicitly_ allowed by `Core.AppliesViaContainer`. We wanted to have this explicit (in order to satisfy the CAP compiler).

Shall we exempt the temporal vocabulary from this rule?

Reply:
Yes, please revert the change to the Temporal vocabulary